config/sidekiq_scheduler.yml

```diff
@@ -3,7 +3,7 @@
 # at the specified interval.
 
 CreateDailySpoolFiles:
-  cron: "0 3 * * * America/New_York"
+  cron: "0 3 * * 1-5 America/New_York"
```

Comment:
This change sets the job to only run on "days of the week 1-5", where 0 is Sunday, and 1 is Monday. Cron is fun.

https://crontab.guru/#0_3_*_*_1-5